Missing various providers values in the `go:build` tag

**Which component are you using?**:

Cluster Autoscaler

**Problem**:
The CA is failed to build with various cloud providers, these providers are `baiducloud`, `cherryservers`, `cloudstack`, `exoscale`, `kamatera`,  `ovhcloud`. It failed due to the missing values of them in the go:build stage under the [builder_all.go](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/builder/builder_all.go) file.

The following error occurred when trying to build CA with baiducloud:

```
$ BUILD_TAGS=baiducloud make build
rm -f cluster-autoscaler-amd64
CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o cluster-autoscaler-amd64 --ldflags "-s" --tags baiducloud
# k8s.io/autoscaler/cluster-autoscaler/cloudprovider/builder
cloudprovider/builder/builder_baiducloud.go:29:5: AvailableCloudProviders redeclared in this block
	cloudprovider/builder/builder_all.go:55:5: other declaration of AvailableCloudProviders
cloudprovider/builder/builder_baiducloud.go:34:7: DefaultCloudProvider redeclared in this block
	cloudprovider/builder/builder_all.go:86:7: other declaration of DefaultCloudProvider
cloudprovider/builder/builder_baiducloud.go:36:6: buildCloudProvider redeclared in this block
	cloudprovider/builder/builder_all.go:88:6: other declaration of buildCloudProvider
make: *** [Makefile:42: build-arch-amd64] Error 2

```
this error comes with other missing cloud providers.

**Proposed Solution**:
Update  `go:build` tag for the missing values for the cloud provider. so that CA will build without any error.

--- a/cluster-autoscaler/cloudprovider/builder/builder_all.go
+++ b/cluster-autoscaler/cloudprovider/builder/builder_all.go
@@ -1,5 +1,5 @@
-//go:build !gce && !aws && !azure && !kubemark && !alicloud && !magnum && !digitalocean && !clusterapi && !huaweicloud && !ionoscloud && !linode && !hetzner && !bizflycloud && !brightbox && !packet && !oci && !vultr && !tencentcloud && !scaleway && !externalgrpc && !civo && !rancher && !volcengine
-// +build !gce,!aws,!azure,!kubemark,!alicloud,!magnum,!digitalocean,!clusterapi,!huaweicloud,!ionoscloud,!linode,!hetzner,!bizflycloud,!brightbox,!packet,!oci,!vultr,!tencentcloud,!scaleway,!externalgrpc,!civo,!rancher,!volcengine
+//go:build !gce && !aws && !azure && !kubemark && !alicloud && !magnum && !digitalocean && !clusterapi && !huaweicloud && !ionoscloud && !linode && !hetzner && !bizflycloud && !brightbox && !packet && !oci && !vultr && !tencentcloud && !scaleway && !externalgrpc && !civo && !rancher && !volcengine && !baiducloud && !cherry && !cloudstack && !exoscale && !kamatera && !ovhcloud
+// +build !gce,!aws,!azure,!kubemark,!alicloud,!magnum,!digitalocean,!clusterapi,!huaweicloud,!ionoscloud,!linode,!hetzner,!bizflycloud,!brightbox,!packet,!oci,!vultr,!tencentcloud,!scaleway,!externalgrpc,!civo,!rancher,!volcengine,!baiducloud,!cherry,!cloudstack,!exoscale,!kamatera,!ovhcloud
 
 /*
 Copyright 2018 The Kubernetes Authors.

